wallet/udb/stakedb.go

```diff
@@ -115,53 +83,23 @@ func maybeConvertDbError(err error) error {
 //
 var (
 	// Bucket names.
-	mainBucketName         = []byte("main")
 	sstxRecordsBucketName  = []byte("sstxrecords")
 	ssgenRecordsBucketName = []byte("ssgenrecords")
 	ssrtxRecordsBucketName = []byte("ssrtxrecords")
-	metaBucketName         = []byte("meta")
+	stakeMetaBucketName    = []byte("meta")
 
 	// Db related key names (main bucket).
-	stakeStoreVersionName    = []byte("stakestorever")
 	stakeStoreCreateDateName = []byte("stakestorecreated")
 )
 
-// int8ToBytes converts an 8 bit signed integer into a 1-byte slice.
-func int8ToBytes(number int8) []byte {
-	var w bytes.Buffer
-	var buf [1]byte
-	buf[0] = byte(number)
-	w.Write(buf[:])
-
-	return w.Bytes()
-}
-
-// int8FromBytes converts a 1 byte byte slice to a signed 8 bit integer.
-func int8FromBytes(slice []byte) int8 {
-	var number int8
-
-	buf := bytes.NewBuffer(slice[:])
-	binary.Read(buf, byteOrder, &number)
-
-	return number
-}
-
-// uint32ToBytes converts a 32 bit unsigned integer into a 4-byte slice in
-// little-endian order: 1 -> [1 0 0 0].
-func uint32ToBytes(number uint32) []byte {
-	buf := make([]byte, 4)
-	byteOrder.PutUint32(buf, number)
-	return buf
-}
-
 // deserializeSStxRecord deserializes the passed serialized tx record information.
 func deserializeSStxRecord(serializedSStxRecord []byte) (*sstxRecord, error) {
 	record := new(sstxRecord)
 
 	curPos := 0
 
 	// Read MsgTx size (as a uint64).
-	msgTxLen := int(byteOrder.Uint64(
+	msgTxLen := int(binary.LittleEndian.Uint64(
```

Comment:
I also just manually verified that the (big endian) byteOrder variable is only used in txdb.go (renamed from wtxmgr/db.go) and for the version in the new metadata bucket.